src/map/pc.cpp

```diff
@@ -3509,8 +3509,7 @@ int pc_setparam(dumb_ptr<map_session_data> sd, SP type, int val)
             }
             break;
         case SP::SEX:
-            // this is a really bad idea
-            sd->sex = static_cast<SEX>(val);
+            chrif_char_ask_name(AccountId(), sd->status_key.name, 5, HumanTimeDiff());
```

Comment:
I wish there was a way to actually send the value of ```set Sex``` to the char server so that there could be more than 2 sexes